File: app/src/main/java/omniex/nl/omniex/ui/app/profile/address/ProfileAddressView.java
package omniex.nl.omniex.ui.app.profile.address;

import java.util.List;

import nl.omniex.omniexshopping.data.model.address.Address;
import nl.omniex.omniexshopping.ui.base.BaseView;

public interface ProfileAddressView extends BaseView {
    void onAddressListFetched(List<Address> addressList);
    void onAddressRemoved();
}

File: app/src/main/java/omniex/nl/omniex/ui/app/profile/address/ProfileAddressView.kt
package omniex.nl.omniex.ui.app.profile.address

import omniex.nl.omniex.data.model.address.Address
import omniex.nl.omniex.ui.base.BaseView

interface ProfileAddressView : BaseView {
    fun onAddressListFetched(addressList: List<Address>)
    fun onAddressRemoved()
}
